app/metrics.yaml

```diff
@@ -8,6 +8,26 @@ no_lint:
   - CATEGORY_GENERIC
 
 events:
+  app_opened_all_startup:
+    type: event
+    description: |
+      A user opened the app to the HomeActivity. The HomeActivity
```

Comment:
"The system received an Intent for the HomeActivity. An intent is received an external entity wants to the app to display content. Intents can be received when the app is closed – at which point the app will be opened – or when the app is already opened – at which point the already open app will make changes such as loading a url. This can be used loosely as a heuristic for when the user requested to open the app. The HomeActivity encompasses..."

Or something like that. We need to make it clear this is not "user opened the app" anymore.

Reply:
Honestly, I would put all those details below what was initially written. Let's get the simple version first so it's easy for someone to understand the high-level, and then move all the details afterwards.

Reply:
@sraturi Can you address this in the follow-up? It should be okay to change the description